mr/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputCommitter.java

```diff
@@ -232,7 +232,7 @@ public void abortJob(JobContext originalContext, int status) throws IOException
           .executeWith(tableExecutor)
           .onFailure((output, exc) -> LOG.warn("Failed cleanup table {} on abort job", output, exc))
```

Comment:
Is this log also missing a placeholder? Seems like it has two arguments, but only one placeholder.

My IDE doesn't seem to complain, but it looks like the same situation you're fixing. Possibly it's not complaining because it's in a lambda? If you could look into that it would be great 👍.

Reply:
For warning, the second arguments of `exc` is an Exception, so it is correct.